### DISCUSSION TITLE
IntelliJ - theme.colors.dark[8] - "unresolved variable dark" warning

### DISCUSSION THREAD

**Discussion by mrclrchtr** [mantinedev/mantine#4670] [Q&A]
Hi, I am new to frontend development and TypeScript and have decided to use Mantine. So far it is actually running very well, but I have the following error message in IntelliJ:

<img width="451" alt="image" src="https://github.com/mantinedev/mantine/assets/16151098/0b155d46-08a6-4fa9-b3bc-08d1be2129a8">

The project and UI is working. So it's probably a wrong message or?

I can't understand if this is the right place to ask about it, or if it rather belongs in an IntelliJ or TypeScript forum. But maybe it belongs to Mantine and someone here can tell me how to solve the problem.

Thank you very much for your help!

**Comment by rtivital** [comment]
Your TypeScript settings are seemingly too strict. Can you send your `tsconfig.json`?

**Reply by mrclrchtr** [reply]
Yes sure:
tsconfig.json
```
{
  "compilerOptions": {
    "target": "ES2020",
    "useDefineForClassFields": true,
    "lib": [
      "ES2020",
      "DOM",
      "DOM.Iterable"
    ],
    "module": "ESNext",
    "skipLibCheck": true,

    /* Bundler mode */
    "moduleResolution": "bundler",
    "allowImportingTsExtensions": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx",

    /* Linting */
    "strict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noFallthroughCasesInSwitch": true
  },
  "include": [
    "src",
  ],
  "references": [
    {
      "path": "./tsconfig.node.json"
    }
  ]
}
```
and tsconfig.node.json
```
{
  "compilerOptions": {
    "composite": true,
    "skipLibCheck": true,
    "module": "ESNext",
    "moduleResolution": "bundler",
    "allowSyntheticDefaultImports": true,
  },
  "include": [
    "vite.config.ts"
  ]
}
```

**Reply by rtivital** [reply]
I do not see anything unusual in the tsconfig. Usually, such issues are caused by this rule – https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature.
Probably the issue is in the editor, try opening the same project in VS Code and see if the issue persists.

**Reply by mrclrchtr** [reply]
Hi, in VS Code, there is no issue. So I also think it's an IntelliJ issue.
Thanks for your help!
